python/cucim/src/cucim/skimage/morphology/tests/test_binary.py

```diff
@@ -65,7 +65,7 @@ def _get_decomp_test_data(function, ndim=2):
         img = cp.zeros((17,) * ndim, dtype=cp.uint8)
         img[(8,) * ndim] = 1
     else:
-        img = cp.asarray(data.binary_blobs(32, n_dim=ndim, seed=1))
+        img = cp.asarray(data.binary_blobs(32, n_dim=ndim, rng=1))
```

Comment:
It looks like [our `binary_blobs` doesn't take `rng` currently]( https://github.com/rapidsai/cucim/blob/5e25641bce2e52595eae160802d02ebd3e6de911/python/cucim/src/cucim/skimage/data/_binary_blobs.py#L4-L6 ). Should we add it?

Reply:
see [4f16f7a](https://github.com/rapidsai/cucim/pull/621/commits/4f16f7a6b4ce544e686fc93127165d77a1849a11) for the additional changes